### ISSUE TITLE
How to output "allMetrics" in Cloud ML Engine hyperparamter tuning jobs?

### ISSUE THREAD

**Issue by rasmi** [googleapis/googleapis#440]
The [docs describe](https://cloud.google.com/ml-engine/reference/rest/v1/projects.jobs#hyperparameteroutput) an `allMetrics` field in the hyperparameter tuning output which contains TensorFlow output values such as loss, accuracy, precision, recall, etc. I found [this](https://github.com/googleapis/googleapis/blob/c07bd5a12e561ebe4e074b218bdd29304cb7a370/google/cloud/ml/v1/job_service.proto#L379) in the repo as well. My output for each tuning trial contains `finalMetric` which in this case is loss, since that is what I configured the hyperparameter trainer to optimize for:
```
"finalMetric": {
    "trainingStep": "12345",
    "objectiveValue": 0.12345
}
```
But it does not contain `allMetrics`, and it is not clear how to make it output any values in `allMetrics`. Is this something I can put in my configuration file or pass as a command line argument when submitting the training job? Any suggestions would be appreciated.

**Comment by rasmi** [comment]
@lukesneeringer any ideas, or any other source you can point me to which may be more helpful? I've also opened an issue on the GoogleCloudPlatform/cloudml-samples repository.

**Comment by rasmi** [comment]
According to the response [here](https://github.com/GoogleCloudPlatform/cloudml-samples/issues/72), there is currently no way to do this.
